src/main/java/core/basesyntax/strategy/impl/BalanceOperationServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationService;
+
+public class BalanceOperationServiceImpl implements OperationService {
```

Comment:
in such case you don't need to add impl, because you have several implementations of such interface, fix it in every implementation